src/components/AccountAssets/AccountAssetsList.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Button, Collapse, Stack, useDisclosure } from '@chakra-ui/react'
+import { CAIP19 } from '@shapeshiftoss/caip'
+import { useMemo } from 'react'
+import { FaArrowCircleDown, FaArrowCircleUp } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
+import { AccountSpecifier } from 'state/slices/portfolioSlice/portfolioSlice'
+
+import { AssetAccountRow } from '../AssetAccounts/AssetAccountRow'
+/**
+ * This returns the assets inside an account
+ * @param assetIds An array of CAIP19s for the account
+ * @param accountId The AccountSpecifier for the account
+ * @param limit If no limit is provided, all assets will be shown. If 0 is provided all assets will go into the more section.
+ * @returns returns JSX
+ */
+
+type AccountAssetListProps = {
+  assetIds: CAIP19[]
+  accountId: AccountSpecifier
+  limit?: number
+}
+export const AccountAssetsList = ({ assetIds, accountId, limit }: AccountAssetListProps) => {
+  const { isOpen, onToggle } = useDisclosure()
+  const translate = useTranslate()
+
+  const featuredAssetIds = useMemo(() => {
+    return limit === 0 || limit
+      ? assetIds && assetIds.length > limit
+        ? assetIds.slice(0, limit)
+        : assetIds
+      : assetIds
+  }, [assetIds, limit])
```

Comment:
nested ternary is hard to read... I think `limit === 0 ||` will make `featuredAssetIds` either be boolean or an array.